### DISCUSSION TITLE
Add checkbox for "multiple" to combobox demo at zagjs website

### DISCUSSION THREAD

**Discussion by anilanar** [chakra-ui/zag#1279] [General]
Would be great to be able to try it out on the website.

**Comment by TylerAPfledderer** [comment]
@anilanar I got you: https://github.com/chakra-ui/zag/pull/1303

And can view the [preview page of the combobox](https://zag-website-git-fork-tylerapfledderer-website-71bb02-chakra-ui.vercel.app/components/react/combobox) that includes that toggle.

**Comment by anilanar** [comment]
Thank you 💯
